application/lib/settings.dart

```diff
@@ -6,21 +6,21 @@ import 'disposition.dart';
 const double kSettingsWidth = 400.0;
 
 class SettingsTab extends StatefulWidget {
-  const SettingsTab({Key key}) : super(key: key);
+  const SettingsTab({Key? key}) : super(key: key);
   @override
   _SettingsTabState createState() => _SettingsTabState();
 }
 
 class _SettingsTabState extends State<SettingsTab> {
-  TextEditingController _username;
-  TextEditingController _password;
+  TextEditingController? _username;
+  TextEditingController? _password;
 
   @override
   void didChangeDependencies() {
     super.didChangeDependencies();
-    _username = TextEditingController(text: ServerDisposition.of(context).username)
+    _username = TextEditingController(text: ServerDisposition.of(context)!.username)
```

Comment:
should ServerDisposition be non-nullable?

Reply:
fixed